mmv1/third_party/terraform/tests/resource_storage_transfer_job_test.go

```diff
@@ -17,12 +17,21 @@ func TestAccStorageTransferJob_basic(t *testing.T) {
 	testUpdatedDataSourceBucketName := randString(t, 10)
 	testUpdatedDataSinkBucketName := randString(t, 10)
 	testUpdatedTransferJobDescription := randString(t, 10)
+	testPubSubTopicName := fmt.Sprintf("topic-%s", randString(t, 10))
```

Comment:
Generally we use the the prefix "tf-test-" for resources created during testing so that our automated sweepers can clean them up if they get abandoned for some reason. Can you change this to "tf-test-topic-%s"?

Reply:
Thanks for pointing it out. I will change it.